### PR TITLE
[ready-for-core-team-review] CRM-16189, fixed code to add entries in civicrm_entity_financial_tr…

### PR DESCRIPTION
**Pull request by pradpnayak**
…xn table when status is changed from partially paid to completed for partially paid contribution associated with Participant

### CODE REVIEW

**Comment by JoeMurray** [comment]
@colemanw please review and merge as a priority. Thanks!

File: CRM/Contribute/BAO/Contribution.php [hunk 1]
```diff
@@ -4816,9 +4816,10 @@ protected static function getRecurringContributionDescription($contribution, $ev
    *
    * @param array $lineItems
    * @param array $contributions
+   * @param array $previousContributionId
    *
    */
-  public static function addPayments($lineItems, $contributions) {
+  public static function addPayments($lineItems, $contributions, $previousContributionId = NULL) {
```
Comment (JoeMurray) [inline CRM/Contribute/BAO/Contribution.php]: Please rename 3rd arg to reflect it is $contributionStatusId

**Comment by colemanw** [comment]
Have done code review and giving "merge on pass" label.
@pradpnayak can you rename the param per @JoeMurray's comment and we can merge this.

**Comment by Edzelopez** [comment]
@colemanw I've renamed the param

**Comment by pradpnayak** [comment]
@yashodha can you merge this please?

**Pull request merged**